### PR TITLE
first NL example set

### PR DESCRIPTION
**Pull request by wasperen2**

### CODE REVIEW

File: NL/2DCode/raw/119.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            {
+                "tg": "840539006",
+                "vp": "",
+                "mp": "",
+                "ma": "",
+                "dn": 4,
+                "sd": 4,
+                "dt": "2021-02-18",
+                "co": "SD",
+                "is": "Ministry of Health Welfare and Sport",
```
Comment (gabywh) [inline NL/2DCode/raw/119.json:21]: Again, syntactically correct - but I wouldn't expect MinVWS to be the issuer for Sudan...
is: this should really be a constrained valueset, but very few Member States have actually agreed what is going to be put there, at least at an EU level

**Review by SchulzeStTSI — APPROVED** [review]

File: NL/2DCode/raw/123.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            {
+                "tg": "840539006",
+                "vp": "1119305005",
+                "mp": "CoronaVac",
+                "ma": "ORG-100032020",
+                "dn": 8,
+                "sd": 9,
+                "dt": "2021-02-18",
+                "co": "GR",
+                "is": "Ministry of Health Welfare and Sport",
```
Comment (gabywh) [inline NL/2DCode/raw/123.json:21]: Again, I suspect MinVWS is not the issuer for GR?

**Review by gabywh — CHANGES_REQUESTED** [review]
1. I didn't eyeball the png
2. I didn't go through in fine detail what is [required] and what is not - best done with tooling in any case - e.g. jsonlogic (but then who checks the checker?)
3. Source code / .json: I made most comments only once even if generally applicable.

File: NL/2DCode/raw/0.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+    "JSON": {
+        "ver": "1.0.0",
+        "nam": {
+            "fn": "Achternaam",
+            "fnt": "ACHTERNAAM",
+            "gn": "Voornaam",
+            "gnt": "VOORNAAM"
+        },
+        "dob": "",
```
[the remark this answers is not in the document]
Comment (ijansch) [reply NL/2DCode/raw/0.json:10]: Noted; testcase was included to see what happens if dob is unknown.

File: NL/2DCode/raw/0.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        "t": [
+            {
+                "tg": "840539006",
+                "tt": "a test",
+                "sc": "2021-04-25T12:45:31Z",
+                "tr": "260415000",
+                "tc": "",
+                "co": "AT",
+                "is": "Ministry of Health Welfare and Sport",
```
[the remark this answers is not in the document]
Comment (ijansch) [reply NL/2DCode/raw/0.json:19]: Will check with DW.

File: NL/2DCode/raw/0.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                "ci": "urn:uvci:01:NL:138bd8d73a7f4d828eb61c46c789c2f5"
```
[the remark this answers is not in the document]
Comment (ijansch) [reply NL/2DCode/raw/0.json:20]: if it's only the checksum, how would a lookup work, if ever the veracity of the data needs to be checked?

File: NL/2DCode/raw/119.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+            "gn": "Voornaam",
+            "gnt": "VOORNAAM"
+        },
+        "dob": "2000-02-29",
+        "v": [
+            {
+                "tg": "840539006",
+                "vp": "",
+                "mp": "",
+                "ma": "",
```
[the remark this answers is not in the document]
Comment (ijansch) [reply NL/2DCode/raw/119.json:16]: Noted.

File: NL/2DCode/raw/104.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
+{
+    "JSON": {
+        "ver": "1.0.0",
+        "nam": {
+            "fn": "Achternaam",
+            "fnt": "ACHTERNAAM",
+            "gn": "Voor-Naam",
+            "gnt": "VOORNAAM"
+        },
+        "dob": "1964-01-XX",
```
[the remark this answers is not in the document]
Comment (ijansch) [reply NL/2DCode/raw/104.json:10]: XX and 00 appear on the passport of dutch people with unknown birthdates. If a check against the passport is made, wouldn't the agent need to see the actual 'XX' to be able to match it with the XX on the passport?

File: NL/2DCode/raw/119.json (hunk 1 above)
Comment (ijansch) [reply NL/2DCode/raw/119.json:21]: What if someone got the vaccination in SD, had their NL doctor confirm the vaccination, and NL doctor gives out the DGC, should there be SD here or NL?

File: NL/2DCode/raw/123.json (hunk 2 above)
Comment (ijansch) [reply NL/2DCode/raw/123.json:21]: MinVWS could be the issuer of a dgc, based on a doctor's vaccination proof, based on a shot actually taken in GR. See above comment.

**Pull request merged**